<!-- use with 640x480 images. got 120Hz with step 5 /-->
<launch>
  <node pkg="nodelet" type="nodeletesm" name="esmnodelet"
args="manager" output="screen"/>

  <node pkg="nodelet" type="nodeletesm" name="ptgrey_simple" args="load
ptgrey_simple/ptgrey_simple esmnodelet" output="screen">
  </node>

  <param name="/image_topic_name" type="string" value="/ptgrey_simple/pgr_mono" />  
  <node pkg="nodelet" type="nodeletesm" name="esmocv" args="load
esmocv/esmocv esmnodelet" output="screen">
    <!--testado fast ddvs (120Hz): lambda=0.8 squareside=250 -->
    <param name="track/maxIte" value="5" />
    <param name="track/mprec" value="2" />
    <!-- if max_RMS is zero, RMS error will not be calculated or checked (saves time) -->
    <param name="track/max_RMS" value="0" />
    <!-- if correlation_window is zero no correlation is used -->
    <param name="track/correlation_window" value="0" />
    <param name="roi/width" value="630" />
    <param name="roi/height" value="470" />
    <!-- if roi/x and roi/y are not set, the rectangle will be centered -->
    <!-- param name="roi/x" value="600" /-->
    <!-- param name="roi/y" value="300" /-->
    <param name="stepmask/step" value="5" />
    <param name="viewscale" value="0.4" />
    <param name="focal_length_pix" value="800" />
  </node>

  <node pkg="nodelet" type="nodeletesm" name="dvsnodelet" args="load
          dvs/ddvs esmnodelet" output="screen">

    <param name="lambda" value="0.41" />
    <param name="ppv" value="1" />

    <param name="robot" value="pioneer" />
    <param name="staubli/host" value="200.144.114.161" />
    <param name="staubli/port" value="11111" />
    <param name="staubli/theta_deg" value ="0" />
  </node>  
  
  <!--
  <remap from = "image" to = "/ptgrey_simple/pgr_mono" />
  <node pkg="nodelet" type="nodeletesm" name="imageview" args="load image_view/image esmnodelet"  output="screen">
    <param name="autosize" value="false" />
  </node>
  -->
  
</launch>
